packages/fluentui/react-telemetry/src/TelemetryTable.tsx

```diff
@@ -174,7 +184,11 @@ export const TelemetryTable: React.FC<TelemetryTableProps> = props => {
           {footerGroups.map(group => (
             <tr {...group.getFooterGroupProps()}>
               {group.headers.find((header: HeaderGroup & { Footer: React.ReactElement }) => header.Footer) &&
-                group.headers.map(column => <td {...column.getFooterProps()}>{column.render('Footer')}</td>)}
+                group.headers.map((column: HeaderGroup & { align?: 'left' | 'right' | 'center' }) => (
```

Comment:
```suggestion
                group.headers.map((column: HeaderGroup & { align?: CellAlign }) => (
```

Let's extract to a type